packages/common/src/api/tan-query/jupiter/utils.ts

```diff
@@ -82,6 +88,15 @@ export async function addTransferFromUserBankInstructions({
     })
 
   instructions.push(secpTransferInstruction, transferInstruction)
+  if (tokenInfo.mintAddress === USDC_MINT_ADDRESS) {
```

Comment:
Is this sufficient? is the only swapping with USDC going to be "internal"?

Reply:
it's the only thing we index as usdc_transaction

Reply:
I guess what I'm asking is are we sure we want to ignore any time we are swapping USDC into something else? my guess is yes, but double checking